roxctl/image/scan/scan_test.go

```diff
@@ -588,3 +588,11 @@ func (s *imageScanTestSuite) runLegacyOutputTests(cases map[string]outputFormatT
 		})
 	}
 }
+
+func (s *imageScanTestSuite) TestScan_IncludeSnoozed() {
+	s.Run("disabled by default", func() {
```

Comment:
nit: no need to use `s.Run()` here, you don't need to create a sub-test for a single test case.